mlir/lib/Quantum/Transforms/merge_rotation.cpp

```diff
@@ -42,37 +42,20 @@ struct MergeRotationsPass : impl::MergeRotationsPassBase<MergeRotationsPass> {
                           << "\n");
 
         Operation *module = getOperation();
-        Operation *targetfunc;
-
-        WalkResult result = module->walk([&](func::FuncOp op) {
-            StringRef funcName = op.getSymName();
-
-            if (funcName != FuncNameOpt) {
```

Comment:
We can remove the target function name option from the tablegen if it is no longer used.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/1323/commits/83fe82f6576ce842e50185557834b2c4f8fa476b